src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -389,7 +382,7 @@ std::string postQueryContext
   {
     ContextElementResponse* cerP  = qcrsP->contextElementResponseVector[ix];
     EntityId*               eP    = &cerP->contextElement.entityId;
-
+    
```

Comment:
Extra whitespace to be removed

(If this was the only one issue in the PR I wouldn't have commented... but given that it would need other changes, I'm doing so :)

Reply:
Yeah, I saw this myself and I have changed it already in my local branch.
This is "important", the style guide checker will complain if not removed.

Reply:
Fixed in 3ed1863